cluster-autoscaler/FAQ.md

```diff
@@ -590,7 +590,7 @@ any nodes left unregistered after this time.
 Every 10 seconds (configurable by `--scan-interval` flag), if no scale-up is
 needed, Cluster Autoscaler checks which nodes are unneeded. A node is considered for removal when **all** below conditions hold:
 
-* The sum of cpu and memory requests of all pods running on this node ([DaemonSet pods](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Mirror pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
+* The maximum value between the sum of cpu and memory requests of all pods running on this node ([DaemonSet pods](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Mirror pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
```

Comment:
suggestion: makes the wording less ambiguous 
```suggestion
* The maximum value between the sum of cpu requests and sum of memory requests of all pods running on this node ([DaemonSet pods](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Mirror pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
```

@faan11 can you also update [the help text for `--scale-down-utilization-threshold`](https://github.com/kubernetes/autoscaler/blob/c068feb5f0e1935bff241f4f2e7cb1f12c763281/cluster-autoscaler/main.go#L123). Would be useful to clarify this in the help text as well.

Reply:
Notes to prove what Fabio said is true : 

https://github.com/kubernetes/autoscaler/blob/57f2814267399e0900a70d567a7813089a3cfa5b/cluster-autoscaler/core/scaledown/eligibility/eligibility.go#L142-L156

https://github.com/kubernetes/autoscaler/blob/b569db410fd8bb152c4cc1a940f059c7e2bf0538/cluster-autoscaler/simulator/utilization/info.go#L61-L78